ZenPacks/zenoss/Import4/events.py

```diff
@@ -122,13 +133,26 @@ def _restoreMySqlDb(self, db):
             cmd_fmt = "cat {sql_path}"
         cmd_fmt += " | {mysql_cmd} {db}"
         cmd = cmd_fmt.format(**locals())
-        proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        proc = subprocess.Popen(cmd, shell=True,
+                                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         while True:
             raw_line = proc.stdout.readline()
             if raw_line != '':
                 self.reportProgress(raw_line)
             else:
                 break
-        self.reportProgress(Results.SUCCESS)
 
-        return Results.SUCCESS
+        proc.wait()
+        if proc.returncode != 0:
+            # report the stderr of the subprocess
```

Comment:
The stderr is redirected to a tmp file. It is processed only when the return code is not successful.